src/schema/v2/index.ts

```diff
@@ -25,7 +25,7 @@ import {
   GravityIDFields,
```

Comment:
How would you feel about renaming this to `v2.ts` and having the index file just `export * from './v2'`?

Reply:
I’m planning on moving things out with better names, but yeah the index will end up being just an export 👍